File: src/main/java/com/ptah/repository/userprofiling/ContactInfo.java
package com.ptah.repository.userprofiling;

import com.ptah.common.BaseEntity;
import jakarta.persistence.Entity;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
public class ContactInfo extends BaseEntity {
    private String name;
    private String address1;
    private String address2;
    private String city;
    private String province;
    private String postalCode;
    private String phoneNumber;
}

File: src/main/kotlin/com/ptah/repository/userprofiling/ContactInfo.kt
package com.ptah.repository.userprofiling

import com.ptah.common.BaseEntity
import jakarta.persistence.Entity
import lombok.Getter
import lombok.Setter

@Entity
@Getter
@Setter
class ContactInfo : BaseEntity() {
    private val name: String? = null
    private val address1: String? = null
    private val address2: String? = null
    private val city: String? = null
    private val province: String? = null
    private val postalCode: String? = null
    private val phoneNumber: String? = null
}
